### DISCUSSION TITLE
Fix Reflection found bug

### DISCUSSION THREAD

**Discussion by schaumb** [stephenberry/glaze#628] [Show and tell]
Hi!
Thanks for the library.
I made a fix for the boost pfr member name reflection. Can you apply it to this repository?
[non-internal linkage](https://github.com/boostorg/pfr/pull/152) 

---

I would be pleased if you could mention anywhere that this reflection method was found by me [here](https://github.com/boostorg/pfr/issues/90).

**Comment by stephenberry** [comment]
Awesome! I will look into this and be sure to credit you. Thanks for sharing!

**Comment by stephenberry** [comment]
@schaumb, this does not appear to work with Clang 15.

**Reply by schaumb** [reply]
It's working. 
https://godbolt.org/z/W9j866Ejv

On clang, you need to wrap member pointer to a class because it's not C++ compliant 
![245125757-a4e7eca2-2a1d-453d-bfa6-515bd26485f0](https://github.com/stephenberry/glaze/assets/6457941/e0c029fe-7a63-4f3c-b993-40e7e30ac822)

**Reply by stephenberry** [reply]
@schaumb Neat, thanks for the compiler explorer link! But, I fail to see how your changes offer any improvement if I need an extern template wrapper? It isn't eliminated my need for an extern template. What would I be gaining?

**Reply by schaumb** [reply]
Currently, you cannot reflect on aggregate types with internal linkages, like function locale classes, unnamed classes, or classes inside the unnamed namespace. My fix enables these classes to reflect.
